src/marshmallow/fields.py

```diff
@@ -1114,7 +1114,7 @@ def _bind_to_schema(self, field_name, schema):
         super()._bind_to_schema(field_name, schema)
         self.format = (
             self.format
-            or getattr(schema.opts, self.SCHEMA_OPTS_VAR_NAME)
```

Comment:
We may want to consider renaming the `schema` arg of `_bind_to_schema` to `parent` to make it a little clearer that this might be a field.

Reply:
Good idea. It's technically a breaking change, so we can do it in 4.0 https://github.com/marshmallow-code/marshmallow/issues/1360